docs/building-experiments/metrics/creating-metrics.md

```diff
@@ -34,14 +34,48 @@ You may actually want to create a metric that is a ratio of two facts. To do thi
 
 In the example above, we're creating a metric that corresponds to the revenue per user per purchase.
 
-5. (Optional) Select a filter
+5. Select a minimum detectable effect
 
-When you created an assignment SQL above, you may have also created additional dimensions, i.e. country or browser. These dimensions are now available under the **Definitions** > **Dimensions SQL**.
+The minimum detectable effect refers to the smallest effect you want to reliably detect in experiments. The lower the minimum detectable effect you set, the more samples you'll need for the experiment to reach conclusive results.
 
-![Select filter](../../../static/img/building-experiments/filter-on-dimensions-create-metric.gif)
+## Metric aggregation types
+
+### Sum
+
+Sum computes metrics that are typically interpreted as averages per entity. If the fact value is NULL, it is imputed as 0.
+
+$\frac{\text{SUM of fact value}}{\text{Number of unique entities}}$
+
+Examples: average revenue per user, sign-up rate, minutes streamed per user, average order value.
+
+### Count Distinct
+
+Count Distinct leverages SQL's ``COUNT DISTINCT`` to compute a unique average per entity. If the fact value is NULL, it is discarded.
+
+$\frac{\text{Sum of (COUNT DISTINCT of fact value for each unique entity)}}{\text{Number of unique entities}}$
+
+Examples: unique videos watched per user, unique articles viewed per visitor, unique products viewed per user.
```

Comment:
It depends a bit on the fact, but I think this is incorrect, usually using count distinct gives you "Number of users who have watched (at least) a video"

Reply:
@schmit I thought that "Number of users who have watched (at least) a video" would require a threshold metric? If I have a fact that lists a unique string associated with each video, and I perform a count distinct over them, couldn't I get numbers higher than 1 for each user?

Reply:
That's a great call-out; your thinking makes a lot of sense but the code disagrees with you:
https://github.com/Eppo-exp/eppo/blob/main/packages/api/src/console/db-aggregations/experiment-subject-summaries.blocks.ts#L76

Reply:
Wow, proven wrong! I'm going to say this is an objectively bad name for this function but document this correctly. It's confusing that:
* `COUNT` is a count of the number of events per user
* `COUNT DISTINCT` is the number of users with an event (rather than distinct events per user)